src/core/loader/ncch.h

```diff
@@ -242,14 +236,22 @@ class AppLoader_NCCH final : public AppLoader {
     u32 core_version = 0;
     u8 priority = 0;
     u8 resource_limit_category = 0;
-    u32 ncch_offset = 0; // Offset to NCCH header, can be 0 or after NCSD header
+
     u32 exefs_offset = 0;
 
-    NCCH_Header ncch_header;
     ExeFs_Header exefs_header;
     ExHeader_Header exheader_header;
 
+protected:
     std::string filepath;
+    // Offset to NCCH header, can be 0 or after NCSD header
+    u32 ncch_offset = 0;
```

Comment:
the way it's currently done looks like the comment applies to the entire block of code, not just to that line.

We also use doxygen to generate automated documentation which depends on specific locations for the comments [which is also why you sometimes see `//<` or `///` which are doxygen commands] - consult the citra style guide foo or existing code for more info